lib/internal/bootstrap/pre_execution.js

```diff
@@ -147,12 +147,10 @@ function setupWarningHandler() {
 
 // https://fetch.spec.whatwg.org/
 function setupFetch() {
-  if (!getOptionValue('--experimental-fetch')) {
+  if (getOptionValue('--no-experimental-fetch')) {
```

Comment:
Do we need this? I think `--no-` CLI flags are understood as negation for boolean flags.

Reply:
need what?

Reply:
Do we need this change?

```console
$ out/Release/node --expose-internals -p 'require("internal/options").getOptionValue("--experimental-fetch")'
true
$ out/Release/node --expose-internals -p 'require("internal/options").getOptionValue("--experimental-fetch")' --no-experimental-fetch
false
```

Reply:
No, we don't need it. I guess you're asking because you prefer the previous version?

Reply:
I think it's clearer to use in the condition the flag that must be passed by the user to change the default behavior, but I can revert the change if you want.